### PR TITLE
events: port core/Death

### PR DESCRIPTION
**Pull request by ackwell**
Title.

This PR:
- Adds all the limit breaks to `ACTION` data because I thought I was going to need the healer ones.
- Tweaks some new event stuff 'cus it's still early days
- Ports core/Death to the new system

### CODE REVIEW

File: src/parser/core/modules/Death.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+		// Queue the raise notification.
+		// Also fabricating a legacy event for backwards compatibility.
+		this.parser.queueEvent({
+			type: 'raise',
+			timestamp: event.timestamp,
+			actor: this.parser.actor.id,
+		})
+		this.parser.fabricateLegacyEvent({
+			type: 'raise',
+			timestamp: event.timestamp,
```
Comment (ackwell) [inline src/parser/core/modules/Death.tsx:129]: akk note; this is inccorrect. event timestamp will be epoch relative, should use `parser.currentTimestamp` or similar

File: src/data/ACTIONS/root/LIMIT_BREAK.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		name: 'Crimson Lotus',
+		icon,
+	},
+
+	// Ranged LB2
+	DESPERADO: {
+		id: 4239,
+		name: 'Desperado',
+		icon,
+	},
```
Comment (ayyaruq) [inline src/data/ACTIONS/root/LIMIT_BREAK.ts:158]: Should this maybe be in the order of LB1, LB2, BRD/MCH/DNC LB3?

File: src/data/STATUSES/root/SHARED.ts [hunk 3]
```diff
@@ -25,6 +26,12 @@ export const SHARED = ensureStatuses({
 		icon: 'https://xivapi.com/i/015000/015011.png',
 	},
 
+	TRANSCENDENT: {
+		id: 418,
+		name: 'Transcendent',
+		icon: 'https://xivapi.com/i/015000/015273.png',
+	},
```
Comment (ayyaruq) [inline src/data/STATUSES/root/SHARED.ts:33]: IIRC this has a max duration of 5s if it matters.

**Review by ayyaruq — COMMENTED** [review]
Looks pretty good, and is a solid example of using the new event system. I do wonder tho, would it be worth also hooking the "you have been given a raise" status and track time where it's sitting there but not accepted/declined? Sometimes you wait for stuff to avoid raising at a bad time (ie in the middle of a protean where you're likely to clip someone) but I figure it's not something that's generally held.

File: src/data/STATUSES/root/SHARED.ts (hunk 3 above)
Comment (ackwell) [reply src/data/STATUSES/root/SHARED.ts:33]: I was hesitant to give it a duration given it drops the moment the player uses something. Duno.

File: src/data/ACTIONS/root/LIMIT_BREAK.ts (hunk 2 above)
Comment (ackwell) [reply src/data/ACTIONS/root/LIMIT_BREAK.ts:158]: Yup. Typo.

**Review by ackwell — COMMENTED** [review]
> Sometimes you wait for stuff to avoid raising at a bad time (ie in the middle of a protean where you're likely to clip someone) but I figure it's not something that's generally held.

this is what i was concerned about. not sure

File: src/data/STATUSES/root/SHARED.ts (hunk 3 above)
Comment (ayyaruq) [reply src/data/STATUSES/root/SHARED.ts:33]: I kinda agree but there's other statuses people can lose early (ie Meditate, Anatman, any other channeling shit), plus actually removing stuff I guess although that's rare outside tanking dungeons with a derpy healer and poor regen timing.

**Pull request merged**